### PR TITLE
Moving o-autoinit, o-brand and o-errors into libraries workspace

### PR DESCRIPTION
**Pull request by JakeChampion**
When making a change to o-autoinit the github ci failed due to percy running on o-autoinit, that's when I noticed it was in the components folder and should be in the libraries folder. I also took this opportunity to move the other libraries which were in the components folder.

During this work I ran the generate workflows script and found that it left behind the old workflows for o-autoinit/brand/errors from when they were in the components folder. I've updated the scripts to remove all the workflows before generating them, that way no old ones are left behind

### CODE REVIEW

**Comment by chee** [comment]
we'll need to rerun ./scripts/generate-workflows on this pr

**Comment by JakeChampion** [comment]
> ./scripts/generate-workflows

That's been done, the npm registry keeps crashing even though it is on the biggest dyno size

**Comment by JakeChampion** [comment]
@chee I tidied up this pull-request, removing the parts which were me testing ideas out and reorganised the commits into a better order

**Comment by JakeChampion** [comment]
I've updated the PR description to reflect the actual changes in this PR now

File: scripts/generate-workflows.js [hunk 1]
```diff
@@ -34,6 +35,8 @@ async function hasScript(workspace, name) {
 	)
 }
 
+await del([".github/workflows/percy-*.yml", ".github/workflows/test-*.yml"])
+
```
Comment (chee) [inline scripts/generate-workflows.js:39]: true

**Review by chee — APPROVED** [review]

**Pull request merged**